SimTKcommon/include/SimTKcommon/internal/ClonePtr.h

```diff
@@ -202,7 +207,11 @@ template <class T> class ClonePtr {
         other.reset(p);
         reset(otherp);
     }
-     
+
+    DEPRECATED_14("use get() instead")
+    const T* getPtr()  const  { return get(); }
+    DEPRECATED_14("use upd() instead")
+    T* updPtr() { return upd(); }
```

Comment:
> I think hiding a method from doxygen, if it exists, can be very frustrating for users.

I think that might be exactly what we are trying to achieve with `[[deprecated]]`. We are trying to tell them not to use the routine, while still allowing them to continue using it if necessary. If they are already using it, they don't need the documentation. If they are thinking about using it, they definitely *should not* do so, so again we shouldn't help them with documentation. Under what circumstance is having the documentation a good thing?

Also there are many public methods and classes that are excluded from doxygen already; we do that when we don't want users to make use of them but have to make them public (or protected) for our own use.